content/en/docs/demo/collector-data-flow-dashboard.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Collector Data Flow Dashboard
+---
+
+Monitoring data flow through the OpenTelemetry Collector is crucial for several
+reasons. Gaining a macro-level perspective on incoming data, such as sample
```

Comment:
macro-level --> high-level?

Reply:
I used macro-level on purpose as later when you have many collectors in place, the board can provide macro-level visibility. As per the demo, I agree, it is just high-level. Shall I change?

Reply:
I don't mind either way. It's easy enough to change later if we need to.